npm2deb/utils.py

```diff
@@ -138,6 +138,9 @@ def create_dir(dir):
 
 def parse_name(name):
     parts = name.partition('@')
+    if parts[0] == '':
```

Comment:
Thanks for the fix.
We could do the same by using `rpartition` method instead of `partition`.

Please check and update.

Reply:
```
+ npm2deb view babel-core
npm reports errors about @babel-core module:
npm ERR! code EINVALIDPACKAGENAME
npm ERR! Invalid package name "@babel-core": name can only contain URL-friendly characters
{
  "error": {
    "code": "EINVALIDPACKAGENAME",
    "summary": "Invalid package name \"@babel-core\": name can only contain URL-friendly characters",
    "detail": ""
  }
}
```

Reply:
changes made